### DISCUSSION TITLE
Website title

### DISCUSSION THREAD

**Discussion by erawhctim** [copper-leaf/ballast#18] [General]
Why does the website say "Orchid" but the library is called "ballast"?

**Comment by cjbrooks12** [comment]
[Orchid](https://github.com/orchidhq/orchid) is the SSG I use to build the website, and that's the default image used there. I just haven't gotten around to making a new image to put there instead.

**Comment by erawhctim** [comment]
Ah! Okay gotcha; I thought maybe it was a prior name you started with for the library.
